### ISSUE TITLE
`computNPA` fails when the input data.frames don't have the same number of rows

### ISSUE THREAD

**Issue by MahShaaban** [philipmorrisintl/NPA#3]
`computNPA` fails with an uninformative error when the number of rows in the `data.frame`s of the input list `dL` is not the same.

``` r
# Load packages
library(NPA)
#> Loading required package: R6
library(NPAModels)

# Load COPD data
data(COPD1)

# Load apoptosis model
net.apopto <- load_model('Mm', 'CFA', 'Apoptosis')

# Score the apoptosis model
npa <- compute_npa(COPD1, net.apopto, verbose = TRUE)
#> Preparing data...
#> Computing differential backbone values...
#> Computing amplitudes of perturbation...
#> Computing variance...
#> Computing downstream reshuffling...
#> Computing backbone reshuffling...
#> Done.

# COPD contain missing values
COPD2 <- COPD1
COPD2$`CS (5m)` <- COPD2$`CS (5m)`[-1,]

compute_npa(COPD2, net.apopto, verbose = TRUE)
#> Preparing data...
#> Error in sapply(dL, function(G) {: non-numeric argument to binary operator
```

<sup>Created on 2021-01-22 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

The function checks the order of `nodeLabel`, but doesn't fail, [here](https://github.com/philipmorrisintl/NPA/blob/9f5829a5732997d4fb9d5143c305e2bd6f4306d9/R/computeNPA.R#L73). Rather it fails when computing `s`, [here](https://github.com/philipmorrisintl/NPA/blob/9f5829a5732997d4fb9d5143c305e2bd6f4306d9/R/computeNPA.R#L91)

**Comment by sgubianpm** [comment]
@MahShaaban, this is correct. I will add some check of the rows of the different slots. Actually, they have to be the same across the slots, so if different, the intersection has to be used.

**Comment by MahShaaban** [comment]
Thanks @sgubianpm 
Yes. To get the function to work, I used the intersection of the slots as you point out.
It would be great if the function can do that automatically, possibly with a warning. Or, the function could fail with an informative error message.
